plugins/node/opentelemetry-hapi-instrumentation/src/hapi.ts

```diff
@@ -148,10 +148,18 @@ export class HapiInstrumentation extends BasePlugin<typeof Hapi> {
     ) {
       if (Array.isArray(pluginInput)) {
         for (const pluginObj of pluginInput) {
-          instrumentation._wrapRegisterHandler(pluginObj.plugin);
+          instrumentation._wrapRegisterHandler(
+            pluginObj.plugin && pluginObj.plugin.plugin
+              ? pluginObj.plugin.plugin
+              : pluginObj.plugin
+          );
         }
       } else {
-        instrumentation._wrapRegisterHandler(pluginInput.plugin);
+        instrumentation._wrapRegisterHandler(
+          pluginInput.plugin && pluginInput.plugin.plugin
```

Comment:
simplify this to
```
pluginInput.plugin?.plugin ?? pluginInput.plugin
```

Reply:
I think I can simplify it with acquiring handler part as well